internal/cac/utils/model.go

```diff
@@ -67,3 +67,89 @@ func CleanPatch(patch models.Rfc7396PatchOperation) {
 	delete(patch, "id")
 	delete(patch, "tenant_id")
 }
+
+func FilterPatch(patch models.Rfc7396PatchOperation, filters []string) (models.Rfc7396PatchOperation, error) {
+	var filterMap = map[string]bool{}
+
+	if len(filters) == 0 {
+		return patch, nil
+	}
+
+	for _, filter := range filters {
+		filterMap[filter] = true
+	}
+
+	if !filterMap["policies"] {
```

Comment:
this code looks like something that could be replaced by a for loop :D

Reply:
almost, there is one exception "scopes" instead of "scopes_without_services" but maybe I can add exception/mapping list instead.

Reply:
two, also "ciba" instead of "ciba_authentication_service"

Reply:
I've added mappings, and refactored it to a simple loop ;)